css/css-ui/outline-017.html

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+a<!DOCTYPE html>
```

Comment:
This looks like a mistake.

Reply:
Indeed. That will teach me to use the online interface to submit my changes :) I probably pressed the "a" a second time after my ctrl+a+delete :)